### DISCUSSION TITLE
Tailwind changes browser styles for autofil

### DISCUSSION THREAD

**Discussion by lessless** [tailwindlabs/tailwindcss#8639] [Help]
Hi,

It seems that Tailwind changes default styles for fields with autofill enabled.

How do I disable that?

**Comment by crswll** [comment]
Can you elaborate on how you think it's changing it? As far as I can see it doesn't do anything special to autofilled fields.

**Comment by lessless** [comment]
I'm still trying to put my finger on where and how this is happening, but here are there screen recordings:

in Chrome text in autofilled inputs is smaller when they are out of focus https://recordit.co/082flqW6LK
in Firefox autofilled input get a lighter shade of yellow and also become semi-transparent https://recordit.co/iRcf61oCGW
